.travis.yml

```diff
@@ -14,7 +14,7 @@ before_install:
 
 script:
   - meson setup build
-  - ninja -C build/
```

Comment:
I would still leave the compile command as a single step. It's easier to see whether the compilation or the test failed.